docs/contributing/style-guide.md

```diff
@@ -34,89 +34,100 @@ Helpful blogs on tech writing:
 * [Feathers](https://ffeathers.wordpress.com/). Blog for technical and fiction writing.
 * [I'd rather be writing](https://idratherbewriting.com/). Guides and thoughts on tech writing process and content.
 
-The Sumo Logic Docs team will review submissions, provide suggested edits, add new content into the navigation, and answer any questions you have.
+The Documentation team will review submissions, provide suggested edits, add new content into the navigation, and answer any questions you have.
 
 :::tip
 To create a new doc quickly, use a template. You can copy and paste the file, add your content, and submit a PR.
-* [Doc topic (generic) template](/docs/contributing/templates/generic-doc)
+* [Doc template](/docs/contributing/templates/generic-doc)
 * [Partner app template](/docs/contributing/templates/partner-app-doc)
 :::
 
 ## Voice and tone
 
-* We are clear, genuine, and trustworthy. We understand that our customers entrust us with their vital data and never make light about our commitment to data access and security.
-* The reader should feel confident and informed. We should strive to engage our customers, and show them where to get additional assistance when needed.
-* Describe Sumo Logic in a professional and truthful manner. Avoid generic, unsubstantial adjectives like "very" or phrases like "we're the best". Instead, illustrate these points by letting our product speak for itself.
-* Avoid using a stiff, institutional voice. Instead, write with an instructive and conversational tone. For example, when linking to the support site, use terms like "Need help? Let us know" instead of "Please email our support personnel" to give our company a friendly face.
-* Instructional content and blog posts should be written at approximately the 8th-grade reading level, particularly in introductory sections, for readability and SEO. You can test your content [here](http://www.writingtester.com).
-* When explaining a process or procedure, clarity is critical. Edit words that distract or confuse. Put yourself into the reader's shoes and think about what actions you are recommended to them when an error message is displayed, rather than merely stating what went wrong. Example: "Could not create the user." vs "This email is already registered in the system. Please use a different email or contact Sumo Logic for assistance."
-* We have a sense of humor! Conveying that we do serious work, but we do not take ourselves too seriously, makes Sumo Logic feel likable.
+* **Clarity and Professionalism**. We are clear, genuine, and trustworthy. We understand that our customers entrust us with their vital data and never make light about our commitment to data access and security.
```

Comment:
Should we use sentence case for the items in this bulleted list? For example, "Clarify and professionalism", "Engagement and assistance", "Professional description", and so on?

Reply:
Oops, good catch! thank you.